### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**
https://github.com/fs/rails_api_format/issues/2

### CODE REVIEW

**Comment by ruslankhaertdinov** [comment]
@timurvafin please take a look

**Review by timurvafin — CHANGES_REQUESTED** [review]
Could you please add sections about Install, and Usage, Credits?

**Comment by ruslankhaertdinov** [comment]
Sure, I'll add them later today.

**Comment by ruslankhaertdinov** [comment]
@timurvafin I remove `Rails API Design` description from Readme, because this gem mostly about error serialization. But was added another requested sections. 
Please take a look when you have a chance.

**Comment by timurvafin** [comment]
> because this gem mostly about error serialization

– not true. Gem changes response on update, create and delete follows specification of REST API Design. Checkout https://github.com/fs/rails-api-format/blob/master/lib/rails_api_format/responder.rb

**Comment by ruslankhaertdinov** [comment]
@timurvafin thank you for the clarification, I'll bring back removed part and update readme.

**Comment by ruslankhaertdinov** [comment]
@timurvafin I updated `Readme` and added link to the [REST API Design](https://github.com/fs/rails-api-format/wiki/REST-API-Design) wiki page

File: README.md [hunk 1]
```diff
@@ -1,32 +1,73 @@
...
+expect(response).to be_an_error_representation(:unauthorized, "Invalid email or password.")
+```
+
+## Getting Help
+
+If you find a bug, please report an [Issue](https://github.com/fs/rails-api-format/issues/new).
+
+## Credits
+
+`Rails API Format` is maintained by [Timur Vafin](http://github.com/timurvafin).
```
Comment (timurvafin) [inline README.md]: Let's say `.. is maintained and was written by Flatstack with the help of our contributors.`

**Review by timurvafin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ruslankhaertdinov) [reply README.md]: @timurvafin done

**Pull request merged**